src/module/helpers/handlebars.ts

```diff
@@ -1,34 +1,42 @@
-import { range } from 'lodash'
+import { compact, range } from 'lodash'
 import { RANKS } from '../constants'
 import { capitalize } from './util'
 import { marked } from 'marked'
 import {
+  ACTION_DICE_ROLLED,
   ACTION_DIE_SIDES,
+  CHALLENGE_DICE_ROLLED,
   CHALLENGE_DIE_SIDES,
-  ROLL_SCORE_MAX,
+  DIE_LOWEST_FACE,
+  SCORE_MAX,
 } from '../rolls/roll.js'
 
 interface RollClassesOptions {
   canceled: boolean
   type: 'action' | 'challenge' | undefined
 }
-function classesForRoll(r, opts?: Partial<RollClassesOptions>) {
+function classesForRoll(
+  r: Roll<RollClassesOptions>,
+  opts?: Partial<RollClassesOptions>
+) {
   const theOpts = {
     ...{ canceled: false, type: undefined },
     ...opts,
   }
   const d = r.dice[0]
-  const maxRoll = d?.faces || ROLL_SCORE_MAX
-  return [
-    d?.constructor.name.toLowerCase(),
-    d && `isiconbg-d${d.faces}-blank`,
-    (d?.total || r.result) <= 1 ? 'min' : null,
-    (d?.total || r.result) == maxRoll ? 'max' : null,
-    theOpts.type,
-    theOpts.canceled ? 'canceled' : null,
-  ]
-    .filter((x) => x)
-    .join(' ')
+  const maxRoll = d?.faces || SCORE_MAX
+  const cssClasses = new DOMTokenList()
```

Comment:
I wasn't reviewing the changes to this file very closely, because it's essentially on its way out the door, but you did break move-rolling with the Ironsworn move sheet here. This is Chrome, which means it'll also be broken in the app.

```
handlebars.ts:28 Uncaught (in promise) TypeError: Illegal constructor
[Detected 1 package: system:foundry-ironsworn]
    at classesForRoll (handlebars.ts:28:22)
```

Can we just drop all the changes to this file that aren't necessary from renames?